app/environments/brassbirmingham/brassbirmingham/envs/test.py

```diff
@@ -17,9 +21,17 @@ def resetGame(self, numPlayers):
 
         if numPlayers > 2:
             self.p3 = Player("Sam", self.board)
+        
+            self.board.tradePosts[0].beerAmount
```

Comment:
What does this do?

Reply:
Gets how many beer are on that trade post. The bonuses aren't done yet and might mean some refactoring for the structure of this

Reply:
But we don't assign it to anything or do anything with this line?

Reply:
Oh good catch yeah I'll remove